apps/imtr/src/parser.ts

```diff
@@ -0,0 +1,182 @@
+import { getAutofillResolverKey } from './autofill.ts';
+
+import { UITV_LOCATION, DMN_REQUIRED_INPUT, DMN_REQUIRED_DECISION, DMNInformationRequirement, RequiredInputOrDecision, DMN_DEFINITIONS, DMNExtensionElement, IMTROption, DMNDecision, DMNDocument, feelTypes, DMNDefinition, DMNInputData, IMTRExecutionRule, DMNInputEntry, DMN_DECISION, DMN_INPUT_DATA, DMN_EXTENSION_ELEMENTS, DMN_INFORMATION_REQUIREMENT, DMN_DECISION_TABLE, DMN_RULE, DMN_OUTPUT_ENTRY, DMN_TEXT, DMN_INPUT_ENTRY, DMN_VARIABLE, UITV_DATA_TYPE, UITV_EXECUTION_RULE, UITV_EXECUTION_RULES, UITV_EXECUTION_RULE_REF, UITV_GEO_REF, UITV_OPTION, UITV_OPTIONS, UITV_OPTION_TEXT, UITV_OPTION_TYPE, UITV_QUESTION, UITV_QUESTION_TEXT, UITV_REUSABLE_ID, CONTENT_CONCLUSION_EXPLANATION, CONTENT_EXECUTION_RULE_EXPLANATION, CONTENT_EXPLANATION, CONTENT_LONG_EXPLANATION, INTER_PRIORITY }
```

Comment:
LOL! Can you not use `import * from './types/imtr.ts';` ?

Reply:
No 😭
I will explain this during the next dev-meeting.
Maybe we can come up with a better solution together.